components/peloton.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Peloton",
+  "owner": ["edwork"],
+  "manifest": "https://raw.githubusercontent.com/edwork/homeassistant-peloton-sensor/master/custom_components/peloton/manifest.json",
```

Comment:
The requirement in the manifest isn't version pinned. Ensure the version is pinned.

Reply:
Hello! I've pinned `pylotoncycle` to `0.2.2` in the repo.

Cheers!